app_grim_reefer/src/testing.c

```diff
@@ -0,0 +1,96 @@
+// Code for testing sensors but stuff that was getting in the way in main
+#include "testing.h"
+
+#include <zephyr/fs/fs.h>
+#include <zephyr/logging/log.h>
+
+LOG_MODULE_REGISTER(grim_test);
+
+#define NUM_SAMPLES 25
+float samples[NUM_SAMPLES] = {0.0f};
+int sample_index = 0;
+float sample_sum = 0;
+
+int samplesi[NUM_SAMPLES] = {0.0f};
+int samplei_index = 0;
+float samplei_sum = 0;
```

Comment:
NIT: Underscores between sample and i

Reply:
leaving my pack rat era behind. resolved because the code no longer exists.